woocommerce/woocommerce-ios
# [Mobile Payments] Play sound and activate Taptic feedback when a payment is completed successfully

Closes #4193 

https://user-images.githubusercontent.com/2722505/119652354-faf51380-bdf3-11eb-9340-1832b0305d07.mov

## Changes
* Play our favourite cha-ching sound when a payment is completed successfully 
* Also trigger haptic feedback for those boring souls, like myself, that always have their devices in silent mode.

## How to test
Testing requires:

1. Setting up a store for Card Present Payment Testing (also P91TBi-4BH-p2 for more specific instructions)
2. WCPay 2.4.0

* Checkout the branch, build and run on device.
* Connect to a reader.
* Hold the device in your hand. Turn the volume up! 🔉 
* Collect a payment for an eligible order. When the process is completed successfully, the device should provide haptic feedback and a sound should be played 💸 

cc @astralbodies 

Update release notes:
